src/d3d9/d3d9_options.h

```diff
@@ -57,8 +57,10 @@ namespace dxvk {
     /// Limit frame rate
     int32_t maxFrameRate;
 
-    /// Set the max shader model the device can support in the caps.
+    /// Set the max shader model the device can support in the caps. these default to d3d9.shaderModel's value if not specifically set  
```

Comment:
I would move the new comment to be right above psShaderModel instead of above shaderModel